default.nix

```diff
@@ -1,12 +1,12 @@
-{ stdenv, nix, lib }:
+{ stdenv, nix, lib, coreutils, direnv }:
 
 stdenv.mkDerivation {
   name = "nix-direnv";
 
   src = ./.;
 
   postPatch = ''
-    sed -i "2iNIX_BIN_PREFIX=${nix}/bin" direnvrc
+    sed -i '2iBIN_PREFIX="${nix}/bin ${coreutils}/bin ${direnv}/bin"' direnvrc
```

Comment:
Since we don't guard against environment variables from the outside, I would rename this to `NIX_DIRENV_BIN`

Reply:
```suggestion
    sed -i '2iNIX_DIRENV_BIN="${nix}/bin ${coreutils}/bin ${direnv}/bin"' direnvrc
```